FirebaseStorageSwift/Tests/Integration/StorageIntegration.swift

```diff
@@ -223,6 +223,20 @@ class StorageIntegration: XCTestCase {
     waitForExpectations()
   }
 
+  func testSimplePutCustomFile() throws {
+    let expectation = self.expectation(description: #function)
+    let fileName = "star_wars.pdf"
+    let bundle = Bundle(for: StorageIntegration.self)
+    let filePath = try XCTUnwrap(bundle.path(forResource: fileName, ofType: ""),
+                                 "Failed to get filePath")
+    let ref = storage.reference(withPath: "ios/public/" + fileName)
+    ref.putFile(from: URL(fileURLWithPath: filePath)) { result in
+      self.assertResultSuccess(result)
+      expectation.fulfill()
+    }
+    waitForExpectations()
```

Comment:
I wonder if this is substantially different from testSimplePutFile(). Do you think we should keep it?

Reply:
Probably not! I was just trying things out with some different file types. I'll remove this test and `star_wars.pdf`. I agree that the current `testSimplePutFile()` is enough here.

I'll also remove the contents of the `Home Improvement.numbers` so we can have a test for attempting to upload a directory.